Yurii_Smazhnyi/3/CryptoSquare.py

```diff
@@ -0,0 +1,42 @@
+import math
+
+def encode(string):
+
```

Comment:
One blank line or no blank lines (if you have docstring) after function definition